astroquery/esa/hubble/core.py

```diff
@@ -179,60 +217,202 @@ def get_postcard(self, observation_id, calibration_level="RAW",
                      "&".join(["?RETRIEVAL_TYPE=POSTCARD",
                                "OBSERVATION_ID=" + observation_id,
                                "CALIBRATION_LEVEL=" + calibration_level,
-                               "RESOLUTION=" + str(resolution)]))
+                               "RESOLUTION=" + str(resolution),
+                               "USERNAME=ehst-astroquery"]))
             log.info(self.copying_string.format(filename))
 
         shutil.move(response, filename)
 
-    def cone_search(self, coordinates, radius=0.0, filename=None,
-                    output_format='votable', cache=True):
+    def cone_search(self, coordinates, radius, filename=None,
+                    output_format='votable', cache=True,
+                    async_job=False, verbose=False):
         """
+        To execute a cone search defined by a coordinate and a radius
+
+        Parameters
+        ----------
+        radius : float
+            radius in arcmin of the cone_search
```

Comment:
This should read 'float or quantity' - if you give a value in degrees, it will be handled correctly be the code below.

Also, `coordinates` is before `radius` in the arguments, so they should be swapped

Reply:
Ok, modified, now we can use a number (in arcmin) or a quantity (in the desired units). Coordinates and radius have also been swapped.

` Parameters
        ----------
        coordinates : astropy.coordinate, mandatory
            coordinates of the center in the cone search
        radius : float or quantity
            radius in arcmin (int, float) or quantity of the cone_search`
            
`        if type(radius) == int or type(radius) == float:
            radius_in_grades = Angle(radius, units.arcmin).deg
        else:
            radius_in_grades = radius.to(units.deg).value`

Reply:
@jespinosaar that change wasn't necessary, `Angle(radius, units.arcmin)` will work if you pass radius as a float or as a quantity:
```python
In [6]: coordinates.Angle(5, unit=u.arcmin)
Out[6]: <Angle 5. arcmin>

In [7]: coordinates.Angle(1/12*u.deg, unit=u.arcmin)
Out[7]: <Angle 5. arcmin>
```

Reply:
I meant for you to only change the documentation, since the code you had before already worked

Reply:
Sorry for the misunderstanding, well, now it will distinguish between cases. Maybe I can change it in the next PR. Thanks!